webhookValidator_test.go

```diff
@@ -20,108 +20,51 @@ package ancla
 import (
 	"errors"
 	"fmt"
-	"net/url"
 	"testing"
+	"time"
 
 	"github.com/stretchr/testify/assert"
 )
 
 var (
-	goodWebhook = Webhook{
-		Config: DeliveryConfig{
-			URL:             "https://www.google.com/",
-			AlternativeURLs: []string{"https://www.google.com/", "https://www.bing.com/"}},
-		FailureURL: "https://www.google.com:1030/software/index.html"}
-	simpleFuncs = []ValidURLFunc{HTTPSOnlyEndpoints(), RejectAllIPs()}
+	positiveFiveDuration, _  = time.ParseDuration("5m")
+	negativeFiveDuration, _  = time.ParseDuration("-5m")
+	positiveThreeDuration, _ = time.ParseDuration("3m")
+	negativeThreeDuration, _ = time.ParseDuration("-3m")
```

Comment:
You can also get these times by multiplying a number by `time.Minute`:
```go
positiveFiveDuration = 5 * time.Minute
```

If you do, you might be able to make them constants?